### ISSUE TITLE
mfiddle embeds broken

### ISSUE THREAD

**Issue by rayshan** [montagestudio/docs.montagestudio.com#3]
E.g. http://docs.montagestudio.com/montagejs/repetition.html

Linking to likely an old nodejitsu link, e.g. http://mfiddle.jit.su/gists/7883458

**Comment by rayshan** [comment]
looks like all mfiddle links are broken, all just show blank starting page

**Comment by sscholle** [comment]
Agreed! All examples listed in the Github "docs.montagestudio.com/montagejs/montagejs-examples.md" fail to load any actual code in Mfiddle.
Please update the links.
